src/rpc/blockchain.cpp

```diff
@@ -2770,6 +2787,7 @@ static RPCHelpMan dumptxoutset()
     CConnman& connman = EnsureConnman(node);
     const CBlockIndex* invalidate_index{nullptr};
     std::unique_ptr<NetworkDisable> disable_network;
+    std::unique_ptr<TemporaryRollback> temporary_rollback;
```

Comment:
Not important, but it would probably make sense for both `disable_network` and  `temporary_rollback` variables to be `std::optional` instead of `std::unique_ptr` since they aren't passed around it would simplify the way they are initialized.